hydroflow_lang/src/graph/ops/repeat_iter.rs

```diff
@@ -17,6 +18,11 @@ pub const REPEAT_ITER: OperatorConstraints = OperatorConstraints {
     is_external_input: false,
     ports_inn: None,
     ports_out: None,
+    properties: FlowProperties {
+        deterministic: FlowPropertyVal::Yes,
+        monotonic: FlowPropertyVal::Yes,
```

Comment:
Probably should be `No` for monotonicity -- if it grows but then repeats itself it can't be monotonic.